1.8/usage/managing-services/index.md

```diff
@@ -1,6 +1,19 @@
 ---
-post_title: Managing Universe Services
+post_title: Managing Services
 menu_order: 70 
 ---
 
-DC/OS Universe services are Marathon applications that are packaged and available from the DC/OS Universe repository. Available DC/OS services include [Mesos frameworks](http://mesos.apache.org/documentation/latest/frameworks/) and other applications. DC/OS services are created by the DC/OS [community](https://dcos.io/community/) and by Mesosphere. 
+DC/OS uses Marathon to manage processes and services and is the "init system" for DC/OS. Marathon starts and monitors your applications and services, automatically healing failures.
```

Comment:
Suggest: "DC/OS uses Marathon to manage processes and services: Marathon is the "init" system for DC/OS.